src/plugins/search-disabled-repos.py

```diff
@@ -71,9 +80,15 @@ def postverifytrans_hook(conduit):
 
     if prompt_permanently_enable_repos(conduit, helpful_new_repos):
         for repo in helpful_new_repos:
-            # FIXME: replace with API calls
-            os.system('subscription-manager repos --enable=%s' % repo)
-            conduit.info(logging.DEBUG, 'Repo permanently enabled: %s' % repo)
+            try:
+                enabled = api.enable_yum_repositories(repo)
+            except Exception:
```

Comment:
> Should we change api.* to raise rhsm.subscription_manager.api specific exceptions itself? So the plugin
> doesn't have to do a blank except?

@alikins Yes, I think that's a good idea.  `except Exception:` can be problematic anyway since it doesn't catch KeyboardInterrupts or SystemExits.

Shall we just merge this PR and then I can go back and create the API exceptions and update the plugin?